spec/controllers/pages_spec.rb

```diff
@@ -0,0 +1,16 @@
+RSpec.describe PagesController, type: :controller do
+
+  describe 'landing page' do
+
+    context 'when it is loaded' do 
+      
+      it 'diplays the landing page' do
```

Comment:
* This test doesn't really assert that it `diplays the landing page`. It merely asserts that the page `loads correctly`. Update the string so the message is truthful

* Try to no commit typos (`diplays`). Self-code-review habit might help